site/static/css/feedback.css

```diff
@@ -0,0 +1,321 @@
+/* Styles for feedback widget */
+/* Thermometer */
+:root {
+    --center-channel-text-64: rgba(61, 60, 64, 0.64);
+    --center-channel-text: rgba(61, 60, 64, 1);
+}
+
+.thermometer-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-end;
+}
+
+.c-thermometer {
+    padding: 2rem 0 0;
+    margin-right: 1rem;
+    margin-bottom: 1rem;
+    display: flex;
+    width: 270px;
+    text-align: center;
+    border-radius: 20px;
+    font-size: 14px;
+}
+
+.c-thermometer__trigger:hover {
+    filter: grayscale(0);
+}
+
+.c-thermometer__popup p {
+    margin: 0 2rem;
+}
+
+.c-thermometer .c-thermometer__paragraph {
+    margin-bottom: 1rem;
+    color: var(--center-channel-text-64);
+}
+
+.c-thermometer__close {
+    cursor: pointer;
+    position: absolute;
+    width: 2rem;
+    height: 2rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #999;
+    font-size: 1.4rem;
+    right: 0.4rem;
+    top: 0.4rem;
+}
+
+.c-thermometer__close:hover {
+    color: #444;
+}
+
+.c-thermometer__emojis {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.c-thermometer__emojis p {
+    margin: 0;
+    font-size: inherit;
+    color: var(--center-channel-text-64);
+}
+
+.c-thermometer__emoji {
+    font-size: 28px;
+}
+
+.c-thermometer__emojis a {
+    z-index: 2;
+    position: relative;
+    color: inherit;
+    text-decoration: none;
+    transition: all 0.2s ease;
+    -moz-transition: all 0.2s ease;
+    -webkit-transition: all 0.2s ease;
+    filter: grayscale(100%);
+    flex: 1;
+}
+
+.c-thermometer__emojis a:hover, .c-thermometer__emojis a.selected {
+    color: inherit;
+    filter: grayscale(0);
+}
+
+@media (max-width: 420px) {
+    .thermometer-container {
+        justify-content: center;
+        align-items: center;
+    }
+    .c-thermometer {
+        margin-right: unset;
+    }
+}
+/* Thermometer ends */
+
+/* Thermometer Modal */
+.c-thermometer-modal__container {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 9999;
+    background: rgb(0 0 0 / 50%);
+}
+
+.c-thermometer-modal__container.show {
+    visibility: visible;
+    opacity: 1;
+    transition: opacity 500ms;
+}
+
+.c-thermometer-modal__container.hide {
+    visibility: hidden;
+    opacity: 0;
+    transition: opacity 500ms;
+}
+
+.c-thermometer-modal__container > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 100%;
+    padding: 0 24px;
+}
+
+.c-thermometer-modal__content {
+    background: white;
+    border-radius: 10px;
+    padding: 22px 45px 45px;
+    max-width: 368px;
```

Comment:
@neflyte Gave everything a look over and everything looks rad - thanks for the lift Alan 🙏

Nitpick: Can we bump this to 400px? The modal container feels a _tad_ narrow in comp to mockups.

Otherwise, that's it from my end~

Reply:
done and done. :smile: